<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ColorStatus</class>
 <widget class="QWidget" name="ColorStatus">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>53</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <item>
    <widget class="QLabel" name="label_0">
     <property name="styleSheet">
      <string notr="true">background-color: rgb(220, 220, 220);</string>
     </property>
     <property name="text">
      <string>Initialization</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="label_1">
     <property name="styleSheet">
      <string notr="true">background-color: rgb(200, 200, 200);</string>
     </property>
     <property name="text">
      <string>Rounds</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="label_2">
     <property name="styleSheet">
      <string notr="true">background-color: rgb(180, 180, 180);</string>
     </property>
     <property name="text">
      <string>Conclusion</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
  </layout>
  <zorder>label_1</zorder>
  <zorder>label_2</zorder>
  <zorder>label_0</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
